src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetFramework.cs

```diff
@@ -24,7 +24,12 @@ private static string ResolveManagedProfilerDirectory()
             var corlib461Version = new Version(corlib461FileVersionString);
             var tracerFrameworkDirectory = corlibVersion < corlib461Version ? "net45" : "net461";
 
-            var tracerHomeDirectory = Environment.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME") ?? string.Empty;
+            var tracerHomeDirectory = Environment.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME");
+            if (string.IsNullOrWhiteSpace(tracerHomeDirectory))
```

Comment:
.NET Framework: Fallback to `DD_INTEGRATIONS` if `DD_DOTNET_TRACER_HOME` was not set.